### DISCUSSION TITLE
update.sh not working

### DISCUSSION THREAD

**Discussion by FoxConsult** [Koenkk/zigbee2mqtt#17703] [Devices]
Hello,

Despite the following packages was updated 7 hours ago on https://github.com/Koenkk/zigbee2mqtt/tree/dev, and I ran the update script more than 24 hours ago, update.sh says "Already up to date.".

"package-lock.json Update zigbee-herdsman-converters to 15.0.115 (#17700) 7 hours ago
package.json Update zigbee-herdsman-converters to 15.0.115 (#17700) 7 hours ago"

I'm running 64bit Bullseye on a Raspberry Pi 4.

What might be the issue?

/Tommie

**Comment by lux73** [comment]
pls search the forum ^^

https://github.com/Koenkk/zigbee2mqtt/discussions/13672

**Reply by FoxConsult** [reply]
Well, that was my post as well and the issue was never resolved.

**Reply by lux73** [reply]
ups, sorry - never used that Script ^^

switch to dev Branch manually -> https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html

but NOT using the Script - **be Aware that the Dev Branch could be unstable!**

_and btw it's never useful to drop a new Thread if there is an old one open_
